spec/factories.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :book do
+    genre { 'Horror' }
```

Comment:
Use [Faker](https://github.com/stympy/faker) to create the factory so it doesn't use the same values always